### ISSUE TITLE
v.4.0.11 Row header is flickering

### ISSUE THREAD

**Issue by m4m4m4** [angular-ui/ui-grid#6480]
Hi, 

I tried out the latest version and noticed the row header flickers when data changes or while scrolling.
It's noticable in both latest chrome and firefox

You'll notice the behaviour in this plunker with tree-view enabled while scrolling:
http://plnkr.co/edit/9bGQISczhzjWILgZ4aok?p=preview

If I load the same plunker with version 4.0.10 the row header doesn't flicker
http://plnkr.co/edit/w5H2GdrItj3akowzCTWf?p=preview

Thanks

- [x] This is a bug report, not a question on how to use the grid.  
Use [Stack Overflow](http://stackoverflow.com/questions/tagged/angular-ui-grid) or [Gitter](https://gitter.im/angular-ui/ui-grid) for questions.
- [x] You have searched the open issues to see if this bug has been filed before.  We don't want duplicate issues.
- [x] You have reviewed the tutorials and documentation to ensure you are using the grid correctly. [ui-grid.info](http://ui-grid.info/docs/#/tutorial)
- [x] You are using the latest version of the grid. Older versions are not patched or supported.
- [x] You have provided steps to recreate your bug.  A plunkr is even better.

**Comment by classix-od** [comment]
I have noticed this problem too. This is because of commit 76e18d6faa9fff5bd866312e3bf1989528d6f9eb.

**Comment by arovetto** [comment]
Any news about this bug? I have the same problem.
Thanks

**Comment by mportuga** [comment]
@AgDude Can you look at this?

**Comment by AgDude** [comment]
I see it in the plunker. I will look into it.

**Comment by arovetto** [comment]
@AgDude, have you understood the cause of the problem?
Thanks

**Comment by AgDude** [comment]
Not fully. I spent some time on it a couple weeks ago, but I have gotten too busy recently to get back to it. I think it has to do with one of the style computations registered with `Grid.registerStyleComputation`. I tried several attempts to address it in the `refreshCanvas` method which didn't work as expected. 

@arovetto if you want to investigate more and provide a PR I will try to find time to review it.
